### PR TITLE
Dual boot solrizer

### PR DESCRIPTION
**Pull request by ShanaLMoore**
This PR is heavily influenced by the [Bulkrax Solrizer solution](https://github.com/samvera-labs/bulkrax/pull/700). 

related to: 
- https://github.com/scientist-softserv/ams/issues/54

[Solrizer](https://github.com/samvera-deprecated/solrizer) is deprecated so calling Solrizer.solr_name breaks for Hyrax 3+. 

Now we are able to access the admin dashboard with dual boot. Previously it caused errors: 

## Rails 6.1:
image

### BEFORE: 

image

### CODE REVIEW

File: app/helpers/solr_helper.rb
Comment (ShanaLMoore) [inline app/helpers/solr_helper.rb:4]: I am not using the dual boot support #rails_5_1? helper method here because that wouldn't necessarily be a valid check.

**Comment by ShanaLMoore** [comment]
Note: this branch is pointing to i54 instead of develop to show the diffs. Since i54's PR wasn't merged, I had to branch this work off of it. I intend to merge this branch into develop once i54 is approved.

**Review by orangewolf — DISMISSED** [review]

**Review by kirkkwang — APPROVED** [review]

**Pull request merged**
